connectivity/manifests/client-egress-to-entities-k8s.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: client-egress-to-entities-k8s
+spec:
+  endpointSelector:
+    matchLabels:
+      kind: client
+  egress:
+  - toEntities:
+    - kube-apiserver
+    toPorts:
+    - ports:
+      - port: "443"
```

Comment:
This isn't the right port; the actual apiserver listens on (usually) 6443. Probably best to leave the port number out.